test/ManagementSpecialPopulations/cases/Young15bLSIL1YearFollowUp.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Younger Than 25 Histologic LSIL At 1 Year Follow Up 1.5b
+
+externalData:
+- resources
+
+data:
+- 
+  resourceType: Patient
+  name: Joanne Smith
+  gender: female
+  birthDate: 2000-01-01
+  extension:
+  -
+    url: http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex
+    valueCode: F
+-  
+  $iterate: *ASCUSOrLSILFourYearsAgo
+- 
+  $iterate: *ASCUSOrLSILThreeYearsAgo
+-  
+  $iterate: *ASCUSOrLSILTwoYearsAgo
+- 
+  $iterate: *HistologyCin1OrNormalLastYear
+
+results:
+  IsIncludedAndNotExcluded: true
+  SpecialPopulationRecommendation: 
+    action: 'LessThan25LSIL5b'
```

Comment:
@mickohanlon23 @ptran265 FYI I'll be updating some of these in the future to correspond to the actual recommended action.